amazon/ion/simpleion.py

```diff
@@ -155,7 +156,56 @@ def dumps(obj, imports=None, binary=True, sequence_as_stream=False,
     return ret_val
 
 
-def load(fp, catalog=None, single_value=True, parse_eagerly=True, text_buffer_size_limit=None):
+class IonPyValueModel(IntEnum):
+    """Flags to control the types of values that are emitted from load(s).
+
+    The flags may be mixed so that users can intentionally demote certain Ion
+    constructs to standard Python types. In general this will increase read
+    performance and improve interoperability with other Python code at the
+    expense of data fidelity.
+
+    For example:
+        value_model = IonPyValueModel.MIXED | IonPyValueModel.SYMBOL_AS_TEXT
+
+    would mean that any Symbols without annotations would be emitted as str,
+    any Symbols with annotations would be emitted as IonPyText(IonType.SYMBOL).
+
+    todo: add/extend this as desired. some possible additions:
+        CLOB_AS_BYTES
+        SEXP_AS______ (list, tuple, either?)
+        IGNORE_ANNOTATIONS
+        TIMESTAMP_AS_DATETIME
+        ALWAYS_STD (union of all flags)
+    """
+
+    ION_PY = 0
+    """All values will be instances of the IonPy classes."""
+
+    MIXED = 1
```

Comment:
Maybe this one could be more descriptive? e.g. `BARE_WHEN_POSSIBLE` or something...

Reply:
Yes! Or something like it. I actually have a revision in my workspace that says this:
```
    MIXED = 1
    """Values will be of the IonPy classes when needed to ensure data fidelity,
    otherwise they will be standard Python types.
    
    todo: I'm not sure about the name or the description... this represents
    the legacy "bare" values boolean. But what _is_ a "bare" value? for some
    types that is the standard Python type. But for a Symbol or a Timestamp
    that is a type defined in amazon.ion.core that is _not_ a "IonPy" type.
    And for struct there is no "bare" type, at least currently.
    """
```